scripts/deploy_site.sh

```diff
@@ -13,6 +13,7 @@ readonly BASEDIR=$(dirname $BASH_SOURCE)
 
 # vulcanize $BASEDIR/../static/elements/elements.html -o $BASEDIR/../static/elements/elements.vulcanized.html --config vulcanize_config.json
 
+compass compile
```

Comment:
It looks like you added a `compass` task as part of the default Gruntfile's default task. Is this separate `compass compile` needed?